src/pages/Auth/SignUpPage/hooks/useFirstErrorMessage.ts

```diff
@@ -0,0 +1,10 @@
+import { useFormContext } from 'react-hook-form';
+import { SignUpFormData } from '../signUpPage/entity';
+
+export default function useFirstErrorMessage() {
+  const { formState: { errors } } = useFormContext<SignUpFormData>();
+
+  const errorList = Object.values(errors);
+
+  return errorList[0]?.message;
```

Comment:
이 훅은 이름을 알아보기도, 반환되는 값도 예상하기 좋아보여요. 👍 
한가지 아쉬운 점은, 함수가 undefined를 반환할 수 있다는 것을 이해하기 어려울 것 같아요.
가독성을 위해 아래처럼 써보는 것은 어떨까요??
```suggestion
  return errorList.length ? errorList[0].message : undefined;
```

Reply:
수정했습니다 ! b8a6102b1de6980312e0f22023608e679ca5c7b9
한가지 궁금한점이 혹시 왜 조건값의 속성을 `length`로 주셨는지 여쭤봐도 될까요..??

Reply:
`Object.values()` 메소드는 객체의 각 프로퍼티 값으로 이루어진 배열을 반환합니다.

error가 없다면 배열의 길이도 0일테니, 배열이 있을 땐 0번째 값을 준다고 명시적으로 표현할 수 있어요.

Reply:
아하... 이해했습니다 감사합니다!